lib/store/index.js

```diff
@@ -288,6 +296,33 @@ export class Store {
     this.kernelMapping.set(newKey, this.kernelMapping.get(oldKey));
     this.kernelMapping.delete(oldKey);
   }
+
+  /**
+   * Returns a kernel attatched to an (multilanguage) ditor
+   */
+  getKernelForEditorBufferPosition(
```

Comment:
Shouldn't kernel return the correct kernel even in markdown files with multi grammars? If not why?

Reply:
As the message of the commit that introduced this function may suggest, this function is needed to support datatip for non-active editors (like the split pane on right) 🦊